RocketChat/Rocket.Chat
# fix: Failed login counter not reset after successful login

## Proposed changes (including videos or screenshots)





## Issue(s)

Closes #26828
## Steps to test or reproduce

- Enable failed log in block thing
- enter your password wrong the defined number of times
- wait for the defined number of minutes to be able to attempt again
- enter the wrong password again
- notice you're now blocked without trying x number of times as set in the setting.

## Further comments



[TC-183]

[TC-183]: https://rocketchat.atlassian.net/browse/TC-183?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
